src/components/Synthetics/PositionItem/PositionItem.tsx

```diff
@@ -273,21 +279,25 @@ export function PositionItem(p: Props) {
   }
 
   function renderLiquidationPrice() {
+    if (!p.position.marketInfo) {
+      return <Skeleton width={60} count={1} baseColor="#B4BBFF1A" highlightColor="#B4BBFF1A" />;
```

Comment:
Does this intended behavior also in mobile? Why does this differ from `...` in other places? 

<img width="580" alt="Screenshot 2024-10-14 at 13 44 39" src="https://github.com/user-attachments/assets/5015be55-e71d-471c-b7b9-ee88c73f55f9">